en-GB/docs/find.ndb

```diff
@@ -369,12 +370,14 @@ spline~Calculates cubic splines for the passed datapoints~csplines ausgleichsfun
 split(STR,CHAR,KEEP)~Splits the string STR at the characters CHAR in separate strings~unterteilen zeichenkette tokenizer zerlegen split()~split(
 sqrt(x)~Square root of x~wurzel root sqrt()~sqrt(
 start~Executes NumeRe scripts (*.nscr), see also "help script"~starten laden plugins installation installieren execute run~help start
+startswith~Gives the value True back, when the second Substring is found at the start of the first string. Otherwise, False.~begin end last erstes letztes wort word front back~startswith(

```

Comment:
```suggestion
startswith(STR,STR)~Returns true, if the second substring is found at the start of the first string. Otherwise, false.~begin end last erstes letztes wort word front back~startswith(
```